### ISSUE TITLE
Flutter plugin of mapmyindia doesn't work with older version of Android

### ISSUE THREAD

**Issue by tarung** [mappls-api/flutter-mapmyindia-gl#9]
In my mobile app I want to load MapMyIndia map so that users can select their current location (or any other location) and their destination using a map. We have decided to use MapMyIndia as the mapping service and Flutter for frontend, we have included the mapmyindia_gl version 0.3.1 plugin for this purpose. The code that we are using :

`Widget mapMyIndiaWidget() {
        return MapmyIndiaMap(
        zoomGesturesEnabled: true,
        compassEnabled: true,
        myLocationEnabled: true,
        tiltGesturesEnabled: true,
        scrollGesturesEnabled: true,
    initialCameraPosition: const CameraPosition(
        target: LatLng(12.972442, 77.580643),
        zoom: 14.0,
    ),
    onMapCreated: (map) => {
        // mapController = map,
    },
    onMapClick: (point, coordinates) {
        print("onMapClick : $coordinates");
    },
    onMapLongClick: (point, coordinates) {
        print("onMapLongClick : $coordinates");
    },
    onMapError: (code, message) {
        print("onMapError : $message");
    },
    );
}
`
This works fine for Android version 12 (API level 31) but when we tried running it for lower Android versions we don't see the map at all (location permissions were granted for this App).

We also tried to downgrade the MMI Flutter to mapmyindia_gl 0.2.0 but we still do not see the Map.

Please suggest how to resolve this issue.

**Comment by tarung** [comment]
This is very high priority for us. Please let us know your initial comments on this.

**Comment by Saksham66** [comment]
@tarung Please Share complete file where you have added that map

**Comment by Tinyhuge** [comment]
> In my mobile app I want to load MapMyIndia map so that users can select their current location (or any other location) and their destination using a map. We have decided to use MapMyIndia as the mapping service and Flutter for frontend, we have included the mapmyindia_gl version 0.3.1 plugin for this purpose. The code that we are using :
> 
> ```
> Widget mapMyIndiaWidget() {
>     return MapmyIndiaMap(
>     zoomGesturesEnabled: true,
>     compassEnabled: true,
>     myLocationEnabled: true,
>     tiltGesturesEnabled: true,
>     scrollGesturesEnabled: true,
> initialCameraPosition: const CameraPosition(
>     target: LatLng(12.972442, 77.580643),
>     zoom: 14.0,
> ),
> onMapCreated: (map) => {
>     // mapController = map,
> },
> onMapClick: (point, coordinates) {
>     print("onMapClick : $coordinates");
> },
> onMapLongClick: (point, coordinates) {
>     print("onMapLongClick : $coordinates");
> },
> onMapError: (code, message) {
>     print("onMapError : $message");
> },
> );
> ```
> 
> }
> 
> This works fine for Android version 12 (API level 31) but when we tried running it for lower Android versions we don't see the map at all (location permissions were granted for this App).
> 
> We also tried to downgrade the MMI Flutter to mapmyindia_gl 0.2.0 but we still do not see the Map.
> 
> Please suggest how to resolve this issue.

Yes, the same issue we are also facing. Expecting the MapmyIndia team to resolve it ASAP.
